README.md

```diff
@@ -13,17 +13,15 @@ A spectral inversion code for retrieval analysis of emission spectra from brown
 ## Current Installation Process for Brewster
 1. If you do not have a fortran compiler on your machine, get the proper version of the gfortran compiler from https://gcc.gnu.org/wiki/GFortranBinaries or https://hpc.sourceforge.net.
 2. Git clone or fork the directory and place wherever your code lives. (If you plan to make any changes to the code you might like to incorporate into the master branch via a pull request, you should probably fork it.)
-3. To keep various package versions clean and organized, you should create a Python 3 environment, ideally Python <=3.11, as that is the version that will be needed to run the code on a cluster. (See list of packages needed in requirements.txt)
-4. You will need to get access to the Linelist and Clouds folders that are shared via Dropbox. Place the Clouds and Linelists folders at the same level as this repository: 
+3. To keep various package versions clean and organized, you should create a Python 3 environment, ideally Python <=3.11, as that is the version that will be needed to run the code on a cluster.
+4. `pip install /path/to/brewster`
```

Comment:
Why not `pip install .` ?

Reply:
Just because I never said `cd brewster_v2` or the specific `git clone` command for that matter

Reply:
I think we need to be super pedantic. I'll push some tweaks. Otherwise, we'll get students saying "/path/to/brewster: path cannot be found"